apidoc/Titanium/UI/WebView.yml

```diff
@@ -272,6 +272,17 @@ methods:
       - name: password
         summary: Basic auth password.
         type: String
+        
+  - name: setRequestHeaders
```

Comment:
should be a property and also return a getter to receive already stored request-headers. Otherwise, I can already see the improvement-ticket coming 😉